src/app/Components/Movie/index.jsx

```diff
@@ -36,12 +48,58 @@ const renderProgressBar = () => {
   );
 };
 
-const Movie = ({ url, title, watchingProgress }) => (
-  <div className="mr-2 mb-4">
-    <img src={url} alt={title} className="max-w-sm max-h-48" />
-    {watchingProgress && renderProgressBar()}
-  </div>
-);
+const Movie = ({ url, title, watchingProgress, id, to, listTitle }) => {
+  const [hovered, setHovered] = useState(false);
+  const dispatch = useDispatch();
+
+  const renderHoveredComponents = () => (
+    <>
+      <img
+        src={addToFavorites}
+        alt="favorites"
+        className="absolute z-10 top-4 left-4 cursor-pointer"
+        onClick={() => {
+          dispatch(addToMyList(id));
+        }}
+      />
+      <img src={shape} alt="shape" className="absolute z-10 top-4 left-10" />
+      <button
+        className={`absolute z-10 font-medium top-4 left-16 text-sm rounded-3xl p-2.5 flex 
+              items-center h-2 ${styles.buttonBorder} ${styles.withoutOutline} ${styles.purple}`}
+      >
+        PG 13
+      </button>
+      <div className="absolute top-20 left-4 z-10 font-medium text-lg">{capitalCase(title)}</div>
+      <div className="absolute top-32 z-10 right-24">
+        <StarRating filledStars={2} halfFilledStars={1} />
+      </div>
+      <div className="absolute top-40 left-4 z-10 font-normal text-sm">Kids</div>
+      <div
+        className="absolute left-12 z-10 rounded-sm bg-white w-4px h-4px"
+        style={{ top: '169px' }}
+      />
+      <div className="absolute top-40 left-16 z-10 font-normal text-sm">Fantasy</div>
+    </>
+  );
+
+  return (
+    <div
+      className="pr-4 pb-4 relative"
+      onMouseEnter={() => setHovered(true)}
+      onMouseLeave={() => setHovered(false)}
```

Comment:
This is not needed, the hover behavior can be handled with `:hover`
Check: https://tailwindcss.com/docs/hover-focus-and-other-states